packages/headless/src/features/facets/facet-set/facet-set-slice.ts

```diff
@@ -176,6 +176,8 @@ export const facetSetReducer = createReducer(
           ...valuesAfter,
         ];
         facetRequest.numberOfValues = facetRequest.currentValues.length;
+        facetRequest.freezeCurrentValues = true;
+        facetRequest.preventAutoSelect = true;
```

Comment:
Added this, at least preventAutoSelect should be true to prevent ML selecting another value.
For freezeCurrentValues it would make sense, since the order is kept line 173-177